src/ex4/server/DB/pokemonsCache.json

```diff
@@ -1 +1 @@
-{}
\ No newline at end of file
+{"12":["Catch butterfree the bug/flying pokemon"],"16":["Catch pidgey the normal/flying pokemon"],"154":["Catch meganium the grass pokemon"],"time":1657106040055,"27,84":["Catch sandshrew the ground pokemon","Catch doduo the normal/flying pokemon"]}
```

Comment:
make sure not to upload cached data as it's user specific.
using git you should upload only an empty file with `{}` or only the folder with no file at all (and make sure that if it doesn't exists the code will create one